gtsam/slam/EssentialMatrixWithCalibrationFactor.h

```diff
@@ -0,0 +1,119 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file EssentialMatrixWithCalibrationFactor.h
```

Comment:
Does it have to be in it’s on file, or can you just be added to the other factor file. You could call it essential matrix factory 3 :-)

Reply:
I am calling it `EssentialMatrixFactor4` as we have 3 factors already.